### PR TITLE
First version of discovery service for drobinki API.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
First version of discovery service for drobinki API.

File: services/discovery-service/build.gradle [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+
+ext {
+	springCloudVersion = 'Finchley.SR1'
+}
+
+dependencies {
+	implementation('org.springframework.cloud:spring-cloud-starter')
+	implementation('org.springframework.cloud:spring-cloud-starter-netflix-eureka-server')
```
Comment (JacekMajchrzak) [inline services/discovery-service/build.gradle]: I dont think we will be using netflix eureka, I think we will use native kubernetes staff: http://blog.christianposta.com/microservices/netflix-oss-or-kubernetes-how-about-both/

**Review by JacekMajchrzak — COMMENTED** [review]
I would use probably a native k8s api gateway https://www.getambassador.io/

File: services/discovery-service/src/main/java/pl/drobinki/api/services/eureka/DiscoveryServiceApplication.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package pl.drobinki.api.services.eureka;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+@EnableEurekaServer
+public class DiscoveryServiceApplication {
```
Comment (JacekMajchrzak) [inline services/discovery-service/src/main/java/pl/drobinki/api/services/eureka/DiscoveryServiceApplication.java:9]: Why you are adding anothre spring boot application here? There is already in first commit DrobinkiApiApplication, you build upon it.

Comment (ghost) [reply services/discovery-service/src/main/java/pl/drobinki/api/services/eureka/DiscoveryServiceApplication.java:9]: For me every single component should be a separate application. I know that for small application it is overhead, but we decided to have a playground. Then we need short session with focus on architecture.

Comment (JacekMajchrzak) [reply services/discovery-service/src/main/java/pl/drobinki/api/services/eureka/DiscoveryServiceApplication.java:9]: If you want create different application, then it should have different repository

**Pull request closed without merging**